pom.xml

```diff
@@ -34,33 +34,30 @@
         <profile>
             <id>scala-2.11</id>
             <properties>
-                <scala.major.version>2.11</scala.major.version>
-                <scala.patch.version>12</scala.patch.version>
+                <scala.binary.version>2.11</scala.binary.version>
+                <scala.version>2.11.12</scala.version>
             </properties>
         </profile>
         <profile>
             <id>scala-2.12</id>
             <properties>
-                <scala.major.version>2.12</scala.major.version>
-                <scala.patch.version>15</scala.patch.version>
+                <scala.binary.version>2.12</scala.binary.version>
+                <scala.version>2.12.15</scala.version>
             </properties>
         </profile>
         <profile>
             <id>scala-2.13</id>
             <properties>
-                <scala.major.version>2.13</scala.major.version>
-                <scala.patch.version>7</scala.patch.version>
+                <scala.binary.version>2.13</scala.binary.version>
+                <scala.version>2.13.8</scala.version>
             </properties>
         </profile>
     </profiles>
     <properties>
         <maven-plugin-api.version>3.8.4</maven-plugin-api.version>
         <maven-plugin-annotations.version>3.6.2</maven-plugin-annotations.version>
         <maven-plugin-plugin.version>3.6.2</maven-plugin-plugin.version>
-        <scalafix.version>0.9.33</scalafix.version>
-        <scala.major.version>2.12</scala.major.version>
-        <scala.patch.version>15</scala.patch.version>
-        <scala.version>${scala.major.version}.${scala.patch.version}</scala.version>
```

Comment:
if you remove this, one won't be able to build "by default", without a specific profile.

Reply:
I reverted the change because I could not get `scalafix.ci` to work any other way, but the build always requires a profile to be successful, because it is required by `scala-cross-maven-plugin`.